src/services/raspberrydevconn/app.py

```diff
@@ -0,0 +1,255 @@
+import io
+import sys
+import logging
+import cherrypy
+import paho.mqtt.client as mqtt
+import smbus
+
+from common.WIOTRestApp import *
+from common.SettingsManager import *
+from common.SettingsNode import *
+from common.RESTBase import RESTBase
+from common.CatalogRequest import *
+from common.WIOThread import WIOThread
+
+
+class RaspberryDevConnAPI(RESTBase):
+    def __init__(self, upperRESTSrvcApp, settings: SettingsNode) -> None:
+        super().__init__(upperRESTSrvcApp, 0)
+        self._catreq = CatalogRequest(self.logger, settings)
+
+        # start the connection with the board. If the board is not connected instance a dummy device
+        try:
+            self.logger.debug(f"ENV ONRASPBERRY = {str(os.environ['ONRASPBERRY'])}")
+            self._onrpi = str(os.environ["ONRASPBERRY"]) == "1"
+        except KeyError:
+            self._onrpi = False
+
+        if self._onrpi:
+            import Adafruit_DHT
+
+            # setup connection to arduino
+            self._bus = smbus.SMBus(settings.arduino.i2c_dev)
+            self._ard_i2c_addr = int(settings.arduino.i2c_addr, 0)
+
+            # setup connection to DHT11
+            self.sensor = Adafruit_DHT.DHT11
+            self.logger.debug(type(self.sensor))
+            self.pin = "4"
```

Comment:
It's better to put this in settings.json, as I did for the ArduinoDevConn

Reply:
Fix in [7df4dd4](https://github.com/thegabriele97/WaterIoT/pull/25/commits/7df4dd4810af591f0ff404364eab6596534b1702)